metaspace_converter/tests/to_anndata_test.py

```diff
@@ -36,9 +36,14 @@ def sm(metaspace_credentials) -> SMInstance:
 @pytest.mark.parametrize(
     ("dataset_id", "database", "fdr", "metadata_as_obs", "add_optical_image"),
     [
+        # Just downloading, metadata as uns
         ("2021-09-03_11h43m13s", ("CoreMetabolome", "v3"), 0.1, False, False),
```

Comment:
Maybe it would make sense to move all the datasets we are using for testing into a dedicated project in METASPACE, to have them in one place and prevent them of getting deleted, if someone ever decides to clean up some data from the group.

Reply:
Not directly related to this PR.

Reply:
Good idea! I have the SpaceM-related datasets in a dedicated [project](https://metaspace2020.eu/project/952c884a-f1de-11e9-9d77-ebd64ac202d6). We could create another project for metaspace-converter (datasets can be contained in multiple projects).